docker/config/env.dist

```diff
@@ -0,0 +1,9 @@
+# This file is for settings that are specific to your docker local development
```

Comment:
what if we just put all `.env` in this dir into gitignore and wrote these docs in a README in there to help avoid accidental commitment?

Reply:
To be clear, this message is meant for the ``.env`` file that gets created from ``env.dist``. The idea is that everything in this file is disabled by default, and someday, if and when a dev needs it, they can go into their ``.env`` and selectively change things. However, as we add things, they go into ``env.dist``, so future devs get those new potential settings. It's not ideal, but simple.

This approach isn't risk free. On one project I've been on, the SRE used these files to populate the production environment, rather than as a template for the prod environment, and copied over some stuff that was meant for development only. I'm not sure how to avoid that entirely, but not checking in the file would help. Comments come in as a distant second as a mitigation strategy.

When an environment file is mentioned in ``docker-compose``, it must be present or ``docker-compose`` fails. If these weren't present or auto-created, then step 1 for the project would be to read some docs about creating a ``.env`` file, with all the options you could put in there.  I think that puts a lot of complexity on a developer before they have any experience with a project. We *might* have to create the ``.env`` file anyway, for CI to work, so there would be an auto-created ``.env`` hidden in the code somewhere.

If I misunderstood your question, maybe a PR would help?